docs-2.0/3.ngql-guide/4.job-statements.md

```diff
@@ -89,6 +89,32 @@ nebula> SUBMIT JOB STATS;
 +------------+
 ```
 
+## SUBMIT JOB DOWNLOAD/INSGEST
+
+`SUBMIT JOB DOWNLOAD HDFS`和`SUBMIT JOB INGEST`命令用于将 SST 文件导入 Nebula Graph。详情参见[导入 SST 文件数据](../nebula-exchange/use-exchange/ex-ug-import-from-sst.md)。
+
+`SUBMIT JOB DOWNLOAD HDFS`语句会下载指定的 HDFS 上的 SST 文件。
+
+`SUBMIT JOB INGEST`语句会将下载的 SST 文件导入图空间。
+
+示例：
+
+```ngql
+nebula> SUBMIT JOB DOWNLOAD HDFS "hdfs://192.168.10.100:9000/tmp";
```

Comment:
nebula> SUBMIT JOB DOWNLOAD HDFS "hdfs://192.168.10.100:9000/tmp";
tmp-> 1.sst

Reply:
tmp is a directory that contains all the`.sst` files.